serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,106 @@
+
+== How to migrate
+
+=== Migrate using Flyway Config
+
+* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `Flyway` properties (default value is `false`):
++
+--
+[source,console]
+----
+quarkus.flyway.migrate-at-start=true
+----
+--
+This will create a schema history table `flyway_schema_history` in your database to track the version of each database, recording in it every versioned migration file applied to build that version.
+
+=== Migrate using Flyway CLI
+* If you want to migrate manually you can use the Flyway migration CLI tool.
+
+** Download Flyway migration command-line tool from the link:{flyway_install_url}[Download Flyway Community Edition] link.
+** Add the following properties in the `/conf/flyway.conf` file:
++
+--
+[source,console]
+----
+flyway.url=jdbc:postgresql://localhost:5432/foobardb
+flyway.user=foo
+flyway.password=bar
+----
+--
+** You can specify these options with commands as well. The Flyway CLI will prompt for the username and password if they are missing in the configuration.
++
+--
+[source,console]
+----
+$ flyway migrate -url=jdbc:postgresql://localhost:5432/foobardb -user=foo -password=bar
+----
+--
+** Download the Kogito DDL zip from link:{kogito_ddl_script_url}[this link here] and extract it and use scripts from `postgresql` folder.
+** You can specify the location of the SQL files that need to be migrated using the `flyway.locations` option. For example,
++
+--
+[source,console]
+----
+flyway.locations=classpath:com.mycomp.migration,database/migrations,filesystem:/sql-migrations,s3:migrationsBucket,gcs:migrationsBucket
+----
+--
+
+=== Manually executing scripts
+You can use the provided SQL scripts in zip to migrate the database by executing them one by one.
+
+* You can find all Kogito DDL scripts by visiting link:{kogito_ddl_script_url}[this link here].
+* Download the zip file of the scripts from version same as your Kogito upgrade.
+* Extract the zip file.
+* Execute the `sql` files from `postgresql` folder.
+
+--
+[source,console]
+----
+> psql -H host -U username -d database_name -a -f create_table.sql
+----
+--
+
+== Baseline migration
+In order to apply migrations to an existing schema. It's necessary to establish the baseline from where Flyway should start executing new migrations. That means, in case your schema already contains a structure and from now on, new changes provided by Kogito should be applied via Flyway integration. In Flyway terms, only migrations above `baselineVersion` will then be applied.
+Using below properties you can initialize flyway schema table.
+
+[tabs]
+====
+Quarkus::
++
+--
+[source,console]
+----
+quarkus.flyway.baseline-on-migrate=true
+quarkus.flyway.baseline-version=1.32.0
+----
+--
+Flyway CLI::
++
+--
+[source,console]
+----
+$ flyway -baselineOnMigrate="true" -baselineVersion="1.32.0" migrate
+----
+--
+====
```

Comment:
as you separated the 2 (quarkus and CLI) in the first section, you should adopt the same structure here and create 2 subsections.

Reply:
@radtriste Earlier we established the two ways we can migrate using the flyway. Here, we are providing just a configuration and how it can be used in both ways. I think it is more convenient to put them together under the same section.

Reply:
My issue is that for quarkus it is a configuration to put and for CLI, it is a command... This could be confusing.
In the first section you separate into 2 subsections. I would do the same here to be logical

Reply:
@radtriste I've updated it.